src/OpenTelemetry.Instrumentation.Owin/Implementation/DiagnosticsMiddleware.cs

```diff
@@ -199,11 +210,34 @@ private static void RequestEnd(IOwinContext owinContext, Exception exception)
 
             activity.Stop();
 
+            if (OwinInstrumentationMetrics.HttpClientDuration.Enabled)
+            {
+                var tags = new TagList
+                {
+                    { SemanticConventions.AttributeHttpMethod, owinContext.Request.Method },
+                    { SemanticConventions.AttributeHttpScheme, owinContext.Request.Scheme },
+                    { SemanticConventions.AttributeHttpStatusCode, owinContext.Response.StatusCode },
+                };
+
+                OwinInstrumentationMetrics.HttpClientDuration.Record(activity.Duration.TotalMilliseconds, tags);
```

Comment:
Q: Would it better to always use the stopwatch for metrics? So the duration doesn't change in subtle ways based on tracing/sampling?

Reply:
The Activity duration is used based on [this line from the spec](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-metrics.md#metric-httpserverrequestduration):

> When this metric is reported alongside an HTTP server span, the metric value SHOULD be the same as the HTTP server span duration.

My original inclination was to always use the stopwatch, but felt this approach was more compliant.